docker-compose.yml

```diff
@@ -88,7 +88,7 @@ services:
                        cub kafka-ready -b kafka:29092 1 20 && \
                        echo Waiting for Confluent Schema Registry to be ready... && \
                        cub sr-ready schema-registry 8081 20 && \
-                       java -cp /usr/share/java/kafka-streams-examples/kafka-streams-examples-5.0.0-standalone.jar \
+                       java -cp /usr/share/java/kafka-streams-examples/kafka-streams-examples-5.2.1-standalone.jar \
```

Comment:
@JimGalasyn : I think this is the right path and filename (`/usr/share/java/kafka-streams-examples/kafka-streams-examples-5.2.1-standalone.jar`) but I haven't had the time to actually verify the Docker packaging/setup -- I just searched and replaced for outdated versions/filenames for the packaged jar.